src/fsharp/FSharp.Build/Microsoft.FSharp.NetSdk.props

```diff
@@ -67,14 +65,35 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
     <FscToolExe Condition="'$(OS)' == 'Unix' and '$(FscToolExe)' == ''">RunFsc.sh</FscToolExe>
   </PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="System.ValueTuple" Version="$(DefaultValueTuplePackageVersion)" >
-      <DefaultInclude>true</DefaultInclude>
-    </PackageReference>
+  <ItemGroup Condition="'$(DisableImplicitSystemValueTupleReference)' != 'true'
+                        and ('$(TargetFrameworkIdentifier)' == '.NETStandard' or '$(TargetFrameworkIdentifier)' == '.NETCoreApp')
+                        and !('$(_TargetFrameworkVersionWithoutV)' >= '2.0' )">
+    <PackageReference Include="System.ValueTuple" Version="$(ValueTupleImplicitPackageVersion)" />
+  </ItemGroup>
+
+  <PropertyGroup>
+    <DefaultValueTuplePackageVersion>4.3.1</DefaultValueTuplePackageVersion>
+    <DefaultfSharpCorePackageVersion>4.3.4</DefaultfSharpCorePackageVersion>
```

Comment:
Nit - casing

Reply:
Yes, `DefaultfSharpCorePackageVersion` is wrong